.github/workflows/sync-to-frontend.yml

```
@@ -0,0 +1,50 @@
+        with:
+          node-version: "18"
+
+      - name: Install Dependencies
+        run: npm install
+
+      - name: Generate Hooks
+        run: |
+          npx bruno-api-typescript generate-hooks -i /tmp/bruno -o ./src/apis
+
+      - name: Create PR
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.9)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)